src/Examine/LuceneEngine/Providers/LuceneIndex.cs

```diff
@@ -1185,8 +1296,11 @@ private IndexWriter WriterFactory(Directory d)
             var writer = new IndexWriter(d, FieldAnalyzer, false, IndexWriter.MaxFieldLength.UNLIMITED);
```

Comment:
I think this is where the examine directory methods should be called.

Reply:
@nzdev good catch :)